src/main/java/seedu/address/logic/commands/ExportCommand.java

```diff
@@ -140,15 +213,24 @@ public void writeToCsvFile(File csvFile, JsonNode jsonTree) throws IOException {
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
 
-        Path jsonFilePath = model.getAddressBookFilePath();
-        File jsonFile = jsonFilePath.toFile();
-        File csvFile = new File(csvFilePath);
+        try {
+            ObservableList<Person> filteredPersonObservableList = model.getFilteredPersonList();
```

Comment:
Nice job using the getFilteredPersonList() command to simplify the implementation of the code!